src/js/_enqueues/lib/emoji-loader.js

```diff
@@ -139,6 +167,41 @@
 		return false;
 	}
 
+	/**
+	 * Determines if the browser properly renders Emoji that Twemoji can supplement.
+	 *
+	 * This is a wrapper for browserSupportsEmoji() which attempts to offload the work to a worker to free up the main
+	 * thread.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @private
+	 *
+	 * @param {string} type Whether to test for support of "flag" or "emoji".
+	 *
+	 * @return {Promise<bool>} True if the browser can render emoji, false if it cannot.
+	 */
+	async function browserSupportsEmojiOptimized(type) {
+		if (typeof OffscreenCanvas !== "undefined") {
+			const blob = new Blob(
+				[
+					emojiSetsRenderIdentically.toString() +
+						browserSupportsEmoji.toString() +
+						`postMessage(browserSupportsEmoji(${JSON.stringify(type)}))`,
```

Comment:
This code is problematic since the function name will change when the code is minimized.

Reply:
Oh, that's a great point. Thanks for catching that.

Reply:
Should be fixed with 44c09ee362